src/components/ImageSearch.vue

```diff
@@ -111,16 +186,17 @@
             query() {
                 let keyword = this.current_keyword;
                 let chunk_id = this.inverted_list[keyword]["buckets"][this.page_index];
+                this.results_end = this.inverted_list[keyword]["count"];
+                this.bucket_count = this.inverted_list[keyword]["buckets"].length;
```

Comment:
That's interesting, would always have used the computed value, but I guess that's fine too